docs/content/integrations/tableau.mdx

```diff
@@ -151,3 +155,34 @@ defs = Definitions.merge(
     marketing_team_workspace.build_defs(),
 )
 ```
+
+### Refresh and materialize Tableau assets
+
+You can use Dagster to orchestrate the refreshment of Tableau workbooks and materialization of Looker sheets and dashboards.
+
+To refresh a Tableau workbook with Dagster, pass its definition by passing `StartWorkbookRefreshRequest` to `TableauCloudWorkspace.build_defs(...)` function.
```

Comment:
I would skip this line and just go straight to the code, as the code will show this exact thing.